JobConfig/digitize/DTSfilter.fcl

```diff
@@ -0,0 +1,39 @@
+#
+#  to filter DTS
+#
+#include "Production/JobConfig/common/prolog.fcl"
+#include "Production/JobConfig/digitize/prolog.fcl"
+# name all processes the same, to mask any provenance in ensemble creation
+process_name: DTSFilter
+source : { module_type : RootInput }
+services : @local::Services.SimAndReco 
+physics : {
+  producers : {
+    @table::Digitize.producers
+    @table::Trigger.producers
+  }
+  filters: {
+    @table::Digitize.filters
+    @table::Trigger.filters
+  }
+  analyzers: { @table::Digitize.analyzers }
+  # define the digitization paths.  Trigger paths are added specific to digitization type
+  DigitizePath : @local::Digitize.DigitizeSequence
+}
+physics.filters.Triggerable.MinNDigis : 13
```

Comment:
Thinking more about it, this override is unnecessary; it only affects the 'Untriggered' stream which isn't being written in this job.  so you can remove this line.